kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java

```diff
@@ -317,6 +281,23 @@ public RootPaths rootPaths() {
   }
 
   @Override
+  public String getPodLog(String namespace, String podName, String containerName, boolean pretty) {
```

Comment:
This isn't at all inline with our beautiful DSL :(

`pods().inNamespace(...).withName(...).logs()` would be better, have to think about how to handle container & prettiness in there.

@iocanel Thoughts?